doc/dev/osd_internals/manifest.rst

```diff
@@ -46,24 +48,31 @@ There are some rough edges we may want to improve:
     do_osd_ops.  This probably isn't desirable, see Snapshots section
     below for some options on how generally to mix these operations
     with snapshots.
+    --> Need to fix
   * Appears to assume that the corresponding section of the object
     does not exist (sets FLAG_MISSING) but does not check whether the
     corresponding extent exists already in the object.
+    --> Is this a problem ? There is a workaround (https://github.com/ceph/ceph/pull/28591)
   * Appears to clear the manifest unconditionally if not chunked,
     that's probably wrong.  We should return an error if it's a
     REDIRECT.
+    --> Need to fix
 
 * TIER_PROMOTE:
 
   * Document control flow in this file.  The CHUNKED case is
     particularly unintuitive and could use some explanation.
+    --> Need to add some explanation
   * SET_REDIRECT clears the contents of the object.  PROMOTE appears
     to copy them back in, but does not unset the redirect or clear the
     reference. Does this not violate the invariant?  In particular, as
     long as the redirect is set, it appears that all operations
     will be proxied even after the promote defeating the purpose.
+    --> Need to fix
   * For a chunked manifest, we appear to flush prior to promoting,
     it that actually desirable?
+    --> This can be resovled at a time the tier agent is modified to flush and
+    evict using manifest.
```

Comment:
I'd really like you to replace these questions with a statement about what is present and what your future plans are.